tools/sidekick/config.json

```diff
@@ -8,6 +8,10 @@
     }
   ],
   "plugins": [
+    {
+      "id": "publish",
+      "confirm": true
+    },
```

Comment:
@rofe Can you confirm that this is the correct way to implement https://github.com/adobe/aem-sidekick/pull/262 ? It would be great to have the confirmation since we don't have a way to test this before going live, as I understand it.

Reply:
@meganthecoder sorry, just seeing this now: fwiw, yes, that is correct.

Reply:
Hi @meganthecoder, I can confirm that is the correct property.